communitiesuk/funding-service-design-assessment-store
# FS-3184: Add "All uploaded documents" functionality

### Change description

- Add route to get theme answers dynamically for all file upload field types

---



### How to test

- Check out this branch
- Call the endpoint via http://localhost:3005/ and an application id
- Observe the JSON response

### Screenshots of UI changes (if applicable)

![image](https://github.com/communitiesuk/funding-service-design-assessment-store/assets/117724519/40b64d23-3e7d-4424-a3de-f57e6a7361d0)

### Related 

- https://github.com/communitiesuk/funding-service-design-assessment/pull/319
- https://github.com/communitiesuk/funding-service-design-fund-store/pull/100
